blockly-core/core/ui/block_space/flyout.js

```diff
@@ -740,6 +769,48 @@ Blockly.Flyout.prototype.filterForCapacity_ = function() {
 };
 
 /**
+ * Update the block counts for limited-quantity workspace blocks
+ * @private
+ */
+Blockly.Flyout.prototype.updateBlockLimits_ = function() {
+  Object.keys(this.blockLimits_)
+      .forEach(function (type) {
```

Comment:
Could use [goog.object.forEach](http://google.github.io/closure-library/api/namespace_goog_object.html#forEach), e.g. `goog.object.forEach(this.blockLimits, function(value, type, object) { object.resetTotal() })`